### PR TITLE
Improve platform-specific nature by adding logos to SDK pages

### PR DESCRIPTION
**Pull request by bbewan**
https://app.clubhouse.io/buddybuild/story/6920/add-clarification-to-the-distinct-ios-android-sdk-pages-so-it-s-more-obvious-which-is-which

Includes:

* Added iOS/Android logos to the platform-specific SDK pages.

* Migration of Android manual integration docs into the SDK chapter, by
  combining it with the iOS manual integration page.

* Improved formatting of manual integration steps.

* Updated TOC to show platform for platform-specific SDK pages.

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://127-bb-docs.surge.sh

File: sdk/integration.adoc [hunk 1]
```diff
@@ -142,37 +139,127 @@ BuddyBuildSDK.setup()
...
+The following permissions allow the SDK to send feedback, as well as to
+collect adb logs in case of app crash:
++
+[source,xml]
+----
+<uses-permission android:name="android.permission.INTERNET"/>
+<uses-permission android:name="android.permission.READ_LOGS"/>
+----
+
+. **Setup the SDK on `Application.onCreate()`**
```
Comment (Palleas) [inline sdk/integration.adoc]: Shouldn't this be `in` since we're adding this in the body of a method?

**Comment by bbdocs** [comment]
See the website here: http://127-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://127-bb-docs.surge.sh

**Pull request merged**